watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination_withRO.py

```diff
@@ -331,6 +335,47 @@ def initialize_system(m):
     return
 
 
+def optimize_operation(m):
+    """
+    Unfixes RO operating conditions and sets solver objective
+        - Operating pressure: 1 - 83 bar
+        - Crossflow velocity: 10 - 30 cm/s
+        - Membrane area: 50 - 5000 m2
+        - Volumetric recovery: 10 - 75 %
+    """
+    desal = m.fs.desalination
+
+    # RO operating pressure
+    desal.P2.control_volume.properties_out[0].pressure.unfix()
+    desal.P2.control_volume.properties_out[0].pressure.setub(
+        8300000
+    )  # pressure vessel burst pressure
+    desal.P2.control_volume.properties_out[0].pressure.setlb(100000)
+
+    # RO inlet velocity
+    desal.RO.velocity[0, 0].unfix()
+    desal.RO.velocity[0, 0].setub(0.3)
+    desal.RO.velocity[0, 0].setlb(0.1)
+
+    # RO membrane area
+    desal.RO.area.unfix()
+    desal.RO.area.setub(5000)
+    desal.RO.area.setlb(50)
+
+    # RO recovery - likely limited by operating pressure
+    desal.RO.recovery_vol_phase[0, "Liq"].unfix()
+    desal.RO.recovery_vol_phase[0, "Liq"].setub(0.99)
+    desal.RO.recovery_vol_phase[0, "Liq"].setlb(0.1)
+
+    m.fs.RO_permeate_quality = Constraint(
+        expr=m.fs.permeate.properties[0].conc_mass_phase_comp["Liq", "TDS"] <= 0.5,
+        doc="Permeate quality must be lower than 0.5 kg/m3 or 500ppm",
```

Comment:
```suggestion
        doc="Permeate quality constraint",
```
@arao53 I think we can do this without a constraint. we could do something like `m.fs.permeate.properties[0].conc_mass_phase_comp["Liq", "TDS"].setub(0.5)`